README.md

```diff
@@ -23,24 +23,32 @@ jobs:
       - uses: chartboost/ruff-action@v1
 ```
 
-Alternatively,
-```
-      - uses: chartboost/ruff-action@v1
-```
-can be included as a step in any other workflow file.
-
+## Advanced Usage
 The Ruff action can be customized via optional configuration parameters passed to Ruff (using `with:`):
 
 - version: Must be a Ruff release available on PyPI. By default, latest release of Ruff. You can pin a version, or use any valid version specifier.
-- args: default, `check`
+- args: You can specify the arguments to pass to the ruff command. By default, it's `check`.
 - src: default, '.'
 
+See [Configuring Ruff](https://github.com/astral-sh/ruff/blob/main/docs/configuration.md) for details
+
+### Use a different ruff version
 ```yaml
 - uses: chartboost/ruff-action@v1
   with:
-    src: "./src"
     version: 0.2.2
-    args: --select B
 ```
 
-See [Configuring Ruff](https://github.com/astral-sh/ruff/blob/main/docs/configuration.md) for details
+### Specify a different source directory
+```yaml
+- uses: chartboost/ruff-action@v1
+  with:
+    src: './src'
+```
+
+### Use `ruff format`
+```yaml
+- uses: chartboost/ruff-action@v1
+  with:
+    args: 'format --check'
```

Comment:
https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules

<img width="856" alt="Screenshot 2024-03-19 at 12 03 30 PM" src="https://github.com/ChartBoost/ruff-action/assets/5032356/078faf18-73d8-4bd4-abf7-72532a8a8852">

Are you sure people would want to use `format --check` [ see docs from link/screenshot ] ... I imagine generally want to use either format or check?  Otherwise -- and I haven't tested -- is the failure message sufficiently easy to debug?  

I'm fine with this, if you think correct, but please think through.  If wanting `format --check`, maybe we link to the doc or make the failure code clear to help people with potential debugging?  Thoughts?

Reply:
Ah ... maybe I missed the development.  ruff formatter is different than linter!  So, ruff format --check, is checking that the file is formatted.  In this case, formatter is roughly a replacement for black?  If that's correct, then indeed this makes sense.

Reply:
I believe was confusing 'format' with 'fix' option.

Reply:
Yes,  Ruff formatter is replacement for black. [The Ruff Formatter - Ruff](https://docs.astral.sh/ruff/formatter/#:~:text=The%20Ruff%20formatter%20is%20an,1.2.)

Reply:
would you put a little not in the docs about that?  I imagine - since I got confused others might as well?  Otherwise, we're great to go.